src/main/java/org/broadinstitute/consent/http/health/SendGridStatus.java

```diff
@@ -0,0 +1,68 @@
+package org.broadinstitute.consent.http.health;
+
+import com.codahale.metrics.health.HealthCheck.Result;
+import com.google.gson.Gson;
+
+public class SendGridStatus {
+    private Object page;
+    private StatusObject status;
+
+    public Object getPage() {
+        return page;
+    }
+
+    public void setPage(Object page) {
+        this.page = page;
+    }
+
+    public String getStatus() {
+        return new Gson().toJson(status);
+    }
+
+    public void setStatus(String indicator, String description) {
+        status = new StatusObject();
+        status.setIndicator(Indicator.valueOf(indicator));
+        status.setDescription(description);
+    }
```

Comment:
In more idiomatic java, one would have a constructor for `StatusObject` that takes the two arguments. Even better, the setter for this should probably take a `StatusObject` to begin with so you don't have to do the manual construction here. The only usages of this method are in tests.
```
    public void setStatus(StatusObject status) {
        this.status = status;
    }
```
I think this is leftover fallout from how this class was originally trying to do too much work when calculating the `Result` object. That refactoring has left some of the less conventional object construction around which means there's a little more cleanup to do.